tekton/images/kind/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Runtime image for common dependencies when running kind tests.
+
+FROM golang:1.14.2-alpine
+
+WORKDIR /kind
+
+RUN GO111MODULE=on go get github.com/google/ko/cmd/ko@v0.6.2
+
+RUN apk add --no-cache \
+  bash curl docker git jq openssl
+
+# Install kubectl and make sure it's available in the PATH.
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && chmod +x ./kubectl && mv ./kubectl /bin
+
+RUN curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.10.0/kind-$(uname)-amd64" && chmod +x ./kind && mv ./kind /bin
```

Comment:
We should try to download those per architecture we can support, aka I would like for us to be able to build this image for amd64, arm64, ppc64le and s390x if possible.

Reply:
We do nightly for amd64, arm64, ppc64le and s390x, so if we can support at least arm64 and ppc64le in addition to amd64 it would be nice.

Reply:
The [PLATFORMS](https://github.com/tektoncd/plumbing/blob/1eb7cb568f84be22ff04fd0b2778b1e9fc5d8833/tekton/cronjobs/dogfooding/images/tkn-nightly/cronjob.yaml#L25) is opaque, so I'll need to spend some time drilling into how this works and how to properly wire it up to the Dockerfile. I can look into add the other architectures in another PR.